Dockerfile.develop

```diff
@@ -0,0 +1,45 @@
+FROM node:10-alpine
```

Comment:
. o ( We should probably bump this to 11 or 12 soon. Something for a different PR )

Reply:
Definitely need to bump it to node-14 now, as we don't support anything older than that.

Reply:
you can use `node:lts-alpine` if you want